### PR TITLE
Credit Card UI changes

### PR DESCRIPTION
**Pull request by faiyaz-shaikh**
Credit Card Example UI changes

### CODE REVIEW

File: lib/credit_card_widget.dart [hunk 1]
```diff
@@ -280,13 +280,16 @@ class _CreditCardWidgetState extends State<CreditCardWidget>
...
         children: <Widget>[
           if (widget.bankName != null && widget.bankName!.isNotEmpty)
-            Padding(
-              padding: const EdgeInsets.only(left: 16, top: 16),
-              child: Text(
-                widget.bankName!,
-                maxLines: 1,
-                overflow: TextOverflow.ellipsis,
-                style: defaultTextStyle,
+            Align(
```
Comment (dhvanit-v-simformsolutions) [inline lib/credit_card_widget.dart:283]: We can make extension for checking null and not empty. Like this 
///Extension on nullable strings
extension NullableStringExtension on String? {
  ///checks if string is null or empty.
  bool get isNullOrEmpty => this == null || (this?.isEmpty ?? false);

  ///checks if string is not null and not empty.
  bool get isNotNullAndNotEmpty => this != null && (this?.isNotEmpty ?? false);
}

File: lib/credit_card_widget.dart [hunk 2]
```diff
@@ -304,22 +307,10 @@ class _CreditCardWidgetState extends State<CreditCardWidget>
...
               ],
             ),
           ),
-          const SizedBox(
-            height: 10,
-          ),
+          const SizedBox(height: 10),
           Expanded(
             child: Padding(
               padding: const EdgeInsets.only(left: 16),
```
Comment (dhvanit-v-simformsolutions) [inline lib/credit_card_widget.dart:316]: Give horizontal padding here and remove padding from line number 326, I guess it will reflect same as expected.

**Review by dhvanit-v-simformsolutions — CHANGES_REQUESTED** [review]
Please make changes

File: lib/credit_card_widget.dart [hunk 3]
```diff
@@ -280,13 +280,16 @@ class _CreditCardWidgetState extends State<CreditCardWidget>
         crossAxisAlignment: CrossAxisAlignment.start,
         children: <Widget>[
           if (widget.bankName != null && widget.bankName!.isNotEmpty)
```
Comment (dhvanit-v-simformsolutions) [inline lib/credit_card_widget.dart:282]: Suggestion : We can make extension for checking null and not empty at same time . Like this,

extension NullableStringExtension on String? {

bool get isNullOrEmpty => this == null || (this?.isEmpty ?? false);

bool get isNotNullAndNotEmpty => this != null && (this?.isNotEmpty ?? false);
}

File: lib/credit_card_widget.dart [hunk 4]
```diff
@@ -276,17 +280,21 @@ class _CreditCardWidgetState extends State<CreditCardWidget>
...
         children: <Widget>[
-          if (widget.bankName != null && widget.bankName!.isNotEmpty)
-            Padding(
-              padding: const EdgeInsets.only(left: 16, top: 16),
-              child: Text(
-                widget.bankName!,
-                maxLines: 1,
-                overflow: TextOverflow.ellipsis,
-                style: defaultTextStyle,
+          if (widget.bankName?.isNotEmpty == true)
```
Comment (DhvanitVaghani) [inline lib/credit_card_widget.dart:287]: You can pass directly condition without comparing it. And nullable variable troubling then add default value false

File: lib/credit_card_widget.dart [hunk 5]
```diff
@@ -276,17 +280,21 @@ class _CreditCardWidgetState extends State<CreditCardWidget>
...
+            Align(
+              alignment: Alignment.topRight,
+              child: Padding(
+                padding: const EdgeInsets.only(right: 16, top: 16),
+                child: Text(
+                  widget.bankName!,
```
Comment (DhvanitVaghani) [inline lib/credit_card_widget.dart:293]: Null check

**Review by DhvanitVaghani — CHANGES_REQUESTED** [review]

**Review by DhvanitVaghani — APPROVED** [review]

**Pull request merged**